lesson1/ind.js

```diff
@@ -0,0 +1,67 @@
+// First - arrow function
```

Comment:
Please use spaces instead of tabs - https://www.sublimetext.com/docs/3/indentation.html
I recommend 
```
"tab_size": 2,
"translate_tabs_to_spaces": true
```